pkg/mock/time.go

```diff
@@ -103,13 +104,13 @@ func getTime(timeType string) string {
 	case TimeStampNsAfterDay:
 		return strconv.FormatInt(currentTime.UnixNano()+HourStampNs*24, 10)
 	case Date:
-		return currentTime.Format("2006-01-02")
+		return fmt.Sprintf("\"%s\"", currentTime.Format("2006-01-02"))
```

Comment:
use `"%s"` or "'%s'" is better. how do you think

Reply:
maybe can do like this
```suggestion
		return fmt.Sprintf(`"%s"`, currentTime.Format("2006-01-02"))
```